doc/apiary/v2/fiware-ngsi-simple-cookbook.apib

```diff
@@ -0,0 +1,872 @@
+
++ Response 204
+
+## Context Entity Attributes [/v2/entities/{entityId}/attrs]
+
+### List all attributes [GET /v2/entities/{entityId}/attrs]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+
++ Response 200 (application/json)
+
+        {
+            "temperature": 21.7,
+            "humidity": 60,
+            "location": {
+                "value":  "41.3763726, 2.1864475",
+                "type": "geo:point",
+                "crs": "WGS84"
+            }
+        }
+
+### Replace all attributes [PUT /v2/entities/{entityId}/attrs]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+
++ Request (application/json)
+
+        {
+            "temperature": 21.7,
+            "relativeHumidity": {
+                "value": 60,
+                "accuracy": 1
+            },
+            "seatsNumber": 12
+        }
+
++ Response 200
+
+### Get a single attribute value as text [GET /v2/entities/{entityId}/attrs/{attrId}{?options}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute to be retrieved.
+    + options: text (optional, string) - Indicates format as text
+
++ Response 200 (text/plain)
+
+        21.7
+
+### Get a single attribute value as JSON [GET /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute to be retrieved.
+
++ Response 200 (application/json)
+
+        {
+            "value": 21.7
+        }
+
+### Get a single attribute value. Not exists [GET /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: mistake (required, string) - Attribute to be retrieved.
+
++ Response 404 (application/json)
+
+        {
+            "code": 4567,
+            "message": "Not found",
+            "description": "The entity does not have such an attribute"
+        }
+
+### Update an attribute value [PUT /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute ID
+
++ Request (text/plain)
+
+        17.9
+
++ Response 200
+
+### Update an attribute value - JSON [PUT /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute ID
+
++ Request (application/json)
+
+        {
+            "value": 25.0,
+            "unitCode": "CEL"
+        }
+
++ Response 200
+
+
+### Remove a single attribute [DELETE /v2/entities/{entityId}/attrs/{attrId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrId: temperature (required, string) - Attribute ID
+
++ Response 204
+
+
+## Context Entity Subscriptions [/v2/entities/{entityId}/subscriptions]
```

Comment:
We haven't discussed the subscriptions, registrations and availability subscriptions in detail yet, but I think is ok to include them as they are know in the cookbook .apib, then adjust the examples as we continue the discussion.

NTC